src/components/base/OakCloudinaryImage/OakCloudinaryImage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { CldImage, CldImageProps } from "next-cloudinary";
+
+import { OakImage, OakImageProps } from "../OakImage";
+
+import { getCloudinaryConfig, getCloudinaryIdFromUrl } from "./cloudinary";
+
+export type OakCloudinaryImageProps = Omit<
+  OakImageProps<typeof CldImage>,
+  "src" | "imageProps"
+> & {
+  /**
+   * Override the global cloudinary config set with `setCloudinaryConfig` for this image
+   */
+  config?: CldImageProps["config"];
+  /**
+   * The cloudinary image id or the full cloudinary URL
+   *
+   * Usually in the format `v1234567890/image.jpg`
+   */
+  cloudinaryId: string;
+};
+
+/**
+ * OakCloudinaryImage wraps OakImage providing responsive images from Cloudinary
+ * based on the `sizes` prop.
+ *
+ * Cloudinary config can be set globally with `setCloudinaryConfig` or per image with the `config` prop.
+ */
+export const OakCloudinaryImage = ({
```

Comment:
I figured a change in tack was needed. The Cloudinary processing should be opt-in since most of our images are UI elements and icons which should already be optimised/served locally.

Adding this wrapper component makes it explicit that the image is expected to be served from Cloudinary with all the bells and whistles of responsive sizing.